Gemfile

```diff
@@ -6,10 +6,13 @@ gem 'spring', :group => :development
 gem 'pg'
 gem 'figaro'
 gem "faraday"
+gem "twitter"
+gem 'whenever', :require => false
 
 group :development, :test do
   gem 'rspec-rails', '~> 3.0'
   gem 'pry'
+  gem 'hashie'
```

Comment:
Its like OpenStruct in that it builds you quick objects from a hash. I used Hashie as opposed to openstruct because the test required an object be made from a nested hash which openstruct cant do.